packages/docs/celo-holder-guide/eth-recovery.md

```diff
@@ -46,6 +46,12 @@ This command will return you with:
 - `privateKey`: the private key associated with your address -- please record this private key on paper and not share with anyone else
 - `publicKey`: the public key associated with your address
 
+***Note***
+
+If the address was generated with another derivationPath or address index (the ethereum's default is `"m/44'/60'/0'/0/0"` which will cover almost every case) you can specify both using the flags `addressIndex`, `changeIndex` and `derivationPath`.
+
+Example for `m/44'/78'/1'/4/23` derivationPath: `celocli account:new --mnemonicPath recovery.txt --derivationPath "m/44'/78'/1'" --changeIndex 4 --addressIndex 23 --node https://forno.celo.org`.
```

Comment:
I would rephrase it as follows (can't use a github suggestion because of the "```" being used both for suggesting and for the content).

For example, for the address corresponding to the path `m/44'/78'/1'/4/23` use: 
```
celocli account:new --mnemonicPath recovery.txt --derivationPath "m/44'/78'/1'" --changeIndex 4 --addressIndex 23 --node https://forno.celo.org
```